src/Artsy/Router/v2/__tests__/makeClientApp.test.tsx

```diff
@@ -0,0 +1,170 @@
+import { SystemContextConsumer } from "Artsy"
+import { createRelaySSREnvironment } from "Artsy/Relay/createRelaySSREnvironment"
+import { createMockNetworkLayer } from "DevTools"
+import { mount } from "enzyme"
+import React from "react"
+import { graphql } from "react-relay"
+import { makeClientApp } from "../makeClientApp"
+
+describe("makeClientApp", () => {
```

Comment:
The following tests are our existing `buildClient/Server` tests, but now pointed at the adaptors -- since the adaptors = generic router + specific reaction code.